src/fondant/executor.py

```diff
@@ -197,35 +197,14 @@ def upload_manifest(self, manifest: Manifest, save_path: t.Union[str, Path]):
         """
         Uploads the manifest to the specified destination.
 
-        If the save_path points to the kubeflow output artifact temporary path,
-        it will be saved both in a specific base path and the native kfp artifact path.
-
         Args:
             manifest: The Manifest object to be uploaded.
             save_path: The path where the Manifest object will be saved.
 
         """
-        is_kubeflow_output = (
-            str(save_path) == "/tmp/outputs/output_manifest_path/data"  # nosec
-        )
-
-        if is_kubeflow_output:
-            # Save to the expected base path directory
-            safe_component_name = self.spec.name.replace(" ", "_").lower()
-            save_path_base_path = (
```

Comment:
Are we then no longer saving the manifest relative to the basepath? We need it for data exploration
https://github.com/ml6team/fondant/issues/368#issuecomment-1700824541

Reply:
Added as task